code/modules/mob/living/carbon/human/species.dm

```diff
@@ -236,6 +236,9 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		QDEL_NULL(wings)
 	if(should_have_wings && !wings)
 		wings = new mutantwings()
+		if(ismoth(C))
+			wings.wing_type = C.dna.features["moth_wings"]
+			wings.flight_level = WINGS_FLIGHTLESS
```

Comment:
Why this?

Reply:
If I don't assign the wing type it'll be 'Plain', and seeing how this proc is used by lings, adminheals and other such things it would be nice if it could restore the original appearance.
And moths have FLIGHTLESS flight level by default.

Reply:
I tested it before, without my PR, ahealing a moth who had their wings surgically removed would actually, somehow, make the wings have null flight level and null wing type.

Reply:
Just make sure it works if they have strong wings as well

Reply:
Added a check for that